### DISCUSSION TITLE
No se puede crear usuarios en el administrador por el problema de "cargos"

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#352] [4. Bug Reports] [answered]
Hola

Quería crear un usuario en el web admin, pero el campo "**Cargo**", se encuentra vacío

![image](https://user-images.githubusercontent.com/61419471/126658215-9208a89b-d1f6-4718-9108-1f55433e49a4.png)

lo iba a ver en compañía y no muestra el listado de cargos

![image](https://user-images.githubusercontent.com/61419471/126658305-01753a01-4a4c-4c71-af55-a80883ccb426.png)

y en la base de datos refleja esto

```
"isActive" : true, 
    "jobs" : [
        "Jefe de la USD", 
        "Analista USD", 
        "Colaboradora USD", 
        "Médico Jefe del Equipo de Coloproctología", 
        "Médico", 
        "Enfermera Universitaria", 
        "Médico Jefe del Equipo de Patología Mamaria", 
        "Matrona CASR", 
        "Jefe Control de Gestión", 
        "Coordinador GES", 
        "Enfermera Coloproctologia", 
        "Enfermera Mama", 
        "Enfermera Gástrico", 
        "Médico Mama", 
        "Médico Gástrico", 
        "Medico Coloproctología", 
        "Referente APS", 
        "Especialista", 
        "Enfermera Broncopulmonar", 
        "Médico Cirujano", 
        "Radiólogo Intervencionista", 
        "Radiólogo", 
        "Medico Oncólogo", 
        "Bot", 
        "Médico Otorrino", 
        "Médico Patólogo Oral", 
        "Sigges", 
        "Médico contralor", 
        "Director CESFAM", 
        "Enfermera, SIGGES"
    ], 
```

gracias =>

@gmoplaza tu ves esto? =>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Efectivamente es un bug conocido, el cual ya se encuentra corregido, por favor informarnos si sigues teniendo inconvenientes.

**Reply by NicolasPLG** [reply]
@guillevalin muchas gracias por la respuesta, en el caso que vuelve a ocurrir levantaré otra discussion con referentes a este discusion
